### ISSUE TITLE
FusionAuth Kubernetes - Postgres,  Pass the database connection string

### ISSUE THREAD

**Issue by khandg** [FusionAuth/fusionauth-issues#2306]
I have installed the fusionauth on DigitalOcean kubernetes via helm charts. Also wanted to connect via DigitalOcean managed Postgresql however, it keeps giving maintenance page. I have already verified that database is accessible from within the kube cluster, so connectivity issue is not there. Also verified that username & password i had input in values.xml are working fine. My only doubt is somehow the connecting string which is being formed by FusionAuth is not correct because DigitalOcean postgres requires `sslmode=require` as a parmeter to pass in the connection string. 
I have already bootstraped the database but still the maintenance page is there.

On the maintenance page iam getting this error:

`The error from the database was [The server requested password-based authentication, but no password was provided by plugin null].`

I am not supplying the super username and password and just supplying the fusionauth credentials.

![image](https://github.com/FusionAuth/fusionauth-issues/assets/72009606/f057c528-cfe3-4bd5-b880-56d6e3a05587)

**Comment by khandg** [comment]
same issue with DigitalOcean mysql as well

**Comment by johnmccalla** [comment]
Same issues here. Did you ever figure it out @khandg?

**Comment by khandg** [comment]
Yes, its kind of hack as i ssh into the pod and then updated the fusion auth config with the DB string directly. Then i refreshed the installation page and it worked.

**Comment by jobannon** [comment]
Just a thought here - -could you set the `DATABASE_URL` or `database.host` in the helm chart to avoid the above scenario?
